src/components/GlobalPreferences/Notifications/ManageNotifications.js

```diff
@@ -0,0 +1,384 @@
+        setIsFetching(false)
+        return subscriptions
+      })
+      .catch(error => {
+        setIsFetching(false)
+        setApiError(error.message)
+      })
+  }, [token])
+
+  useEffect(() => {
+    if (!token) return
+    fetchSubscriptions()
+  }, [fetchSubscriptions, token])
+
+  return (
+    <React.Fragment>
+      {apiError && (
+        <Info
+          css={`
+            margin-bottom: ${GU}px;
+          `}
+          mode="error"
+        >
+          {apiError}
+        </Info>
+      )}
+      <Split
+        primary={
+          <SubscriptionsForm
+            onApiError={setApiError}
+            onCreate={fetchSubscriptions}
+            dao={dao}
+            apps={apps}
+            token={token}
+            isFetchingSubscriptions={isFetching}
+            subscriptions={subscriptions}
+          />
+        }
+        secondary={
+          <React.Fragment>
+            <Box heading="Signed In With Email">
+              {email}
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+                wide
+                onClick={onLogout}
+              >
+                Sign Out
+              </Button>
+            </Box>
+            <DeleteAccount
+              onApiError={setApiError}
+              token={token}
+              onLogout={onLogout}
+            />
+          </React.Fragment>
+        }
+      />
+      {!isFetching && subscriptions.length > 0 && (
+        <SubscriptionsTable
+          apps={apps}
+          authToken={token}
+          subscriptions={subscriptions}
+          onUnsubscribe={fetchSubscriptions}
+        />
+      )}
+    </React.Fragment>
+  )
+}
+
+ManageNotifications.propTypes = {
+  apps: PropTypes.arrayOf(AppType).isRequired,
+  dao: PropTypes.string,
+  email: PropTypes.string,
+  onLogout: PropTypes.func,
+  token: PropTypes.string,
+}
+
+function DeleteAccount({ token, onLogout, onApiError }) {
+  const [isFetching, setIsFetching] = useState(false)
+  const [isAccountDeleted, setIsAccountDeleted] = useState(false)
+  const theme = useTheme()
+
+  const handleDeleteAccount = useCallback(async () => {
+    try {
+      setIsFetching(true)
+      await deleteAccount(token)
+      localStorage.removeItem(NOTIFICATION_SERVICE_TOKEN_KEY)
+      localStorage.removeItem(NOTIFICATION_SERVICE_EMAIL_KEY)
+      setIsAccountDeleted(true)
+      onLogout()
+    } catch (e) {
+      onApiError(e.message)
+    }
+    setIsFetching(false)
+  }, [onLogout, token, onApiError])
+
+  return (
+    <Box heading="Email Notification Data">
+      <Button onClick={handleDeleteAccount}>
+        {isFetching ? (
+          <LoadingRing />
+        ) : isAccountDeleted ? (
+          <IconCheck />
+        ) : (
+          <IconTrash
+            css={`
+              color: ${theme.negative};
+              margin-right: ${GU}px;
+            `}
+          />
+        )}
+        Delete your email
```

Comment:
Maybe we should make this button `wide` too and center the text? cc @dizzypaty 

![image](https://user-images.githubusercontent.com/3072458/63693325-15abea00-c814-11e9-9c7e-ec557708792b.png)